Assets/Labrys/Editor/FeatureEditor/Tool.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEditor;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Labrys.Editor.FeatureEditor
+{
+	public abstract class Tool
+	{
+		public string Name { get; protected set; }
+
+		protected EditorWindow window;
+
+		public Tool(EditorWindow window)
```

Comment:
public -> protected (can't use the public constructor anyways)

Reply:
Done.